src/Module/Conversation/Community.php

```diff
@@ -0,0 +1,334 @@
+			DI::page()['aside'] .= TrendingTags::getHTML(self::$content);
+		}
+
+		$t = Renderer::getMarkupTemplate("community.tpl");
+		return Renderer::replaceMacros($t, [
+			'$content' => $o,
+			'$header' => '',
+			'$show_global_community_hint' => (self::$content == 'global') && DI::config()->get('system', 'show_global_community_hint'),
+			'$global_community_hint' => DI::l10n()->t("This community stream shows all public posts received by this node. They may not reflect the opinions of this node’s users.")
+		]);
+	}
+
+	/**
+	 * Computes module parameters from the request and local configuration
+	 *
+	 * @param array $parameters
+	 * @throws HTTPException\BadRequestException
+	 * @throws HTTPException\ForbiddenException
+	 */
+	protected static function parseRequest(array $parameters)
+	{
+		if (DI::config()->get('system', 'block_public') && !Session::isAuthenticated()) {
+			throw new HTTPException\ForbiddenException(DI::l10n()->t('Public access denied.'));
+		}
+
+		self::$page_style = DI::config()->get('system', 'community_page_style');
+
+		if (self::$page_style == CP_NO_INTERNAL_COMMUNITY) {
+			throw new HTTPException\ForbiddenException(DI::l10n()->t('Access denied.'));
+		}
+
+		switch ($parameters['accounttype'] ?? '') {
+			case 'person':
+				self::$accounttype = User::ACCOUNT_TYPE_PERSON;
+				break;
+			case 'organisation':
+				self::$accounttype = User::ACCOUNT_TYPE_ORGANISATION;
+				break;
+			case 'news':
+				self::$accounttype = User::ACCOUNT_TYPE_NEWS;
+				break;
+			case 'community':
+				self::$accounttype = User::ACCOUNT_TYPE_COMMUNITY;
+				break;
+			default:
+				self::$accounttype = null;
+				break;
+		}
+
+		self::$content = $parameters['content'] ?? '';
+		if (!self::$content) {
+			if (!empty(DI::config()->get('system', 'singleuser'))) {
+				// On single user systems only the global page does make sense
+				self::$content = 'global';
+			} else {
+				// When only the global community is allowed, we use this as default
+				self::$content = self::$page_style == CP_GLOBAL_COMMUNITY ? 'global' : 'local';
+			}
+		}
+
+		if (!in_array(self::$content, ['local', 'global'])) {
+			throw new HTTPException\BadRequestException(DI::l10n()->t('Community option not available.'));
+		}
+
+		// Check if we are allowed to display the content to visitors
+		if (!local_user()) {
+			$available = self::$page_style == CP_USERS_AND_GLOBAL;
+
+			if (!$available) {
+				$available = (self::$page_style == CP_USERS_ON_SERVER) && (self::$content == 'local');
+			}
+
+			if (!$available) {
+				$available = (self::$page_style == CP_GLOBAL_COMMUNITY) && (self::$content == 'global');
+			}
+
+			if (!$available) {
+				throw new HTTPException\ForbiddenException(DI::l10n()->t('Not available.'));
+			}
+		}
+
+		// check if we serve a mobile device and get the user settings accordingly
+		if (DI::mode()->isMobile()) {
+			self::$itemsPerPage = DI::pConfig()->get(local_user(), 'system', 'itemspage_mobile_network', 20);
```

Comment:
Introducing a constant for the value to find/adapt it more easily?

Reply:
Yes, but where?

Reply:
Maybe as `Pager::DEFAULT_COUNT` and `Pager::DEFAULT_COUNT_MOBILE` or something similar?

Reply:
The `20` should be part of the default configuration in `static/defaults.config.php`.

Reply:
Look at that:
```php
if (DI::mode()->isMobile()) {
	$itemspage_network = DI::pConfig()->get(local_user(), 'system', 'itemspage_mobile_network');
	$itemspage_network = ((intval($itemspage_network)) ? $itemspage_network : 20);
} else {
	$itemspage_network = DI::pConfig()->get(local_user(), 'system', 'itemspage_network');
	$itemspage_network = ((intval($itemspage_network)) ? $itemspage_network : 40);
}
```
It already exists!

Reply:
Should it be a node-wide setting?

Reply:
I think the default should be node wide.

Reply:
Done.